src/mp-instance.ts

```diff
@@ -222,6 +222,48 @@ export default function mParticleInstance(this: IMParticleWebSDKInstance, instan
     };
 
     this._resetForTests = function(config, keepPersistence, instance) {
+        if (typeof window !== 'undefined') {
```

Comment:
Let's move this additional resetForTests functionality into the config/setup in the test folder to avoid increasing the size of the SDK for test purposes.